app/views/notifications/_reaction.html.erb

```diff
@@ -21,5 +21,5 @@
       </span>
     </div>
 <% else %>
-  <%= render "aggregated_reactions", notification: notification %>
+  <%= render "aggregated_reactions", notification: notification, siblings: siblings %>
```

Comment:
Already defined in line 2 of file: 
`
<% siblings = notification.json_data["reaction"]["aggregated_siblings"].select{|n| n["created_at"] > 24.hours.ago } %>
`